src/app/teachertoolbar/teachertoolbar.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#" style="color:rgb(236, 160, 60); font-size: 40px">Number</a>
+        <a class="navbar-brand" href="#" style="color:rgb(230, 189, 101); font-size: 25px; padding: 9px">Ninja</a>
+      </div>
+      <ul class="nav navbar-nav">
+        <li><a href="#" style="color:yellow">MANAGE ASSIGNMENTS</a></li>
+        <li><a href="#" style="color:yellow">GRADES</a></li>
+      </ul>
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="#"><span class="glyphicon glyphicon-user" style="color:cyan"></span> Log Out</a></li>
+      <!---- <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>-->
```

Comment:
Remove comment in line 13 and try to move the styles to css file like the previous comment.

Reply:
Removed comment. Cannot move styles to the CSS file ( please refer to the previous comment)